lib/services/keyVault/lib/models/jsonWebKey.js

```diff
@@ -149,7 +150,7 @@ JsonWebKey.prototype.mapper = function () {
         },
         k: {
           required: false,
-          serializedName: 'K',
+          serializedName: 'k',
```

Comment:
Seems ok to me, just checking.

---
In reply to: [76497637](https://github.com/Azure/azure-sdk-for-node/pull/1858#discussion_r76497637) [](ancestors = 76497637)